src/cpp/benders/benders_core/BendersBase.cpp

```diff
@@ -202,7 +200,7 @@ void BendersBase::UpdateStoppingCriterion() {
   else if (_data.lb + _options.ABSOLUTE_GAP >= _data.best_ub)
     _data.stopping_criterion = StoppingCriterion::absolute_gap;
   else if (((_data.best_ub - _data.lb) /
-            (std::max)(std::abs(_data.best_ub), std::abs(_data.lb))) <=
+            std::max(std::abs(_data.best_ub), std::abs(_data.lb))) <=
```

Comment:
Is there no conflict from Windows's min/max macros ?

Reply:
Not to my knowledge.  We don't directly include windows.h headers

Reply:
Indeed, CI fail to build on windows : error C2589: '(': illegal token on right side of '::'